packages/rum-core/src/boot/preStartRum.ts

```diff
@@ -187,11 +188,13 @@ export function createPreStartStrategy(
     },
 
     startDurationVital(vitalStart) {
-      bufferApiCalls.add((startRumResult) => startRumResult.startDurationVital(vitalStart))
+      return createVitalInstance((vital) => {
+        bufferApiCalls.add((startRumResult) => startRumResult.addDurationVital(vital))
+      }, vitalStart)
     },
 
-    stopDurationVital(vitalStart) {
-      bufferApiCalls.add((startRumResult) => startRumResult.stopDurationVital(vitalStart))
+    addDurationVital: (vital) => {
+      bufferApiCalls.add((startRumResult) => startRumResult.addDurationVital(vital))
     },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It could be simplied a bit as follow:
```ts
 const addDurationVital = (vital: DurationVitalAdd) => {
    bufferApiCalls.add((startRumResult) => startRumResult.addDurationVital(vital))
  }
  return {
    ...
    addDurationVital,
    startDurationVital(vitalStart) {
      return createVitalInstance((vital) => addDurationVital(vital), vitalStart)
    }
  }
```